README.md

```diff
@@ -117,6 +119,10 @@ withdraw_confirm = { gas = 3000000, gas_price = 1000000000 }
 - `home/foreign.poll_interval` - specify how often home node should be polled for changes (in seconds, default: **1**)
 - `home/foreign.request_timeout` - specify request timeout (in seconds, default: **3600**)
 - `home/foreign.password` - path to the file containing a password for the validator's account (to decrypt the key from the keystore)
+- `home/foreign.gas_price_oracle_url` - the URL used to query the current gas price for the home and foreign nodes.
+- `home/foreign.gas_price_timeout` - the number of seconds to wait for an HTTP response from the gas price oracle before using the default gas price.
+- `home/foreign.gas_price_speed_type` - use the gas price from the oracle URL corresponding to this transaction speed.
+- `home/foreign.default_gas_price` - the default gas price in GWEI used in transactions with the home or foreign node. The default gas price is used when the gas price oracle cannot be URL cannot be reached.
 
 #### authorities options
```

Comment:
It would make sense to show default value here (15.0)

Reply:
Ok, no problem.